bucket/dbvis.json

```diff
@@ -1,36 +1,37 @@
 {
-    "version": "13.0.5",
+    "version": "14.0.1",
     "description": "A universal database tool for developers, DBAs and analysts.",
     "homepage": "https://www.dbvis.com/",
     "license": {
         "identifier": "Freeware",
         "url": "https://www.dbvis.com/eula/"
     },
     "suggest": {
-        "JDK": [
-            "java/oraclejdk",
+        "JDK/JRE": [
+            "java/temurin-jre",
             "java/openjdk"
```

Comment:
separate them or simply drop JDK part.

Reply:
Ok I will when I can, just currently busy with personal stuff at the moment.

Reply:
sure, good day.

Reply:
Done.